src/plot/series/line-series-canvas.js

```diff
@@ -20,7 +20,7 @@
 import PropTypes from 'prop-types';
 import {rgb} from 'd3-color';
 import * as d3Shape from 'd3-shape';
-
+import React from 'react';
```

Comment:
nit: include line after external imports and before internal imports

Reply:
ok will insert line under external imports, i imported react because it was throwing errors